rancher/install-rancher.sh

```diff
@@ -3,16 +3,17 @@
 
 set -x
 
-if [ "$#" -ne 4 ]
+if [ "$#" -ne 5 ]
 then
-	echo "Usage $0 <hostname> <email> <clproject> <apikey>"
+	echo "Usage $0 <hostname> <email> <clproject> <apikey> <bootstrappassword>"
 	exit 1
 fi
 
 HOST=$1
 EMAIL=$2
 PROJECT=$3
 APIKEY=$4
+BOOTSTRAPPASSWORD=$5
```

Comment:
tolkar detta som att vi vill sätta det utifrån och då det behöver skickas in [cloud-config.yaml](https://github.com/glesys/one-click-installers/blob/master/rancher/cloud-config.yaml) också. behöver även implementeras i Cloud i så fall, innan denna PR går ut.

Reply:
Det stämmer! :) Det blir en femte parameter som måste skickas med till shellscriptet Tillagt i shellscript nu.